ios/FluentUI/Command Bar/CommandBarItem.swift

```diff
@@ -124,15 +124,6 @@ open class CommandBarItem: NSObject {
         }
     }
 
-    /// A string that uniquely identifies the element, typically for automation purposes
-    @objc public var accessibilityIdentifier: String? {
```

Comment:
this can't be right. is yours or fluent2 branch up to date?

Reply:
Seems that the fluent2-tokens branch hasn't merged with main with this change, I'll revert it